ui/app/styles/components/selectable-card.scss

```diff
@@ -20,4 +20,8 @@
     width: 7rem;
     min-height: 8rem;
   }
+
+  &.card-width-20 {
+    width: 20rem;
+  }
```

Comment:
I made it REM intentionally so it would increase proportionately - but since this is a new way to handle width's I'll definitely be circling back to make sure the bump doesn't negatively affect these cards.

That being said the text within the cards will also get bigger - which I kept in mind while building this but the end result may still feel too large.